tests/FSharp.Data.GraphQL.Tests/Variables and Inputs/InputScalarAndAutoFieldScalarTests.fs

```diff
@@ -214,3 +215,55 @@ let ``Execute handles nullable auto-fields in input and output object fields coe
         empty errors
         data |> equals (upcast expected)
 
+
+open FSharp.Data.GraphQL.Tests.OptionalsNormalizationTests
+
+[<RequireQualifiedAccess>]
+module ConsoleLoginProviders =
+
+    let [<Literal>] Microsoft365 = "microsoft365"
+    let [<Literal>] GoogleWorkspace = "google_workspace"
+
+type ConsoleLoginProvider =
+    | [<JsonName(ConsoleLoginProviders.Microsoft365)>] Microsoft365
+    | [<JsonName(ConsoleLoginProviders.GoogleWorkspace)>] GoogleWorkspace
+and ApplicationTenantId = ValidString<InputRecord>
+and WrongInput = {
+    Id : ApplicationTenantId
+    /// Legal entity name
+    Name : string
+    LoginProvider : ConsoleLoginProvider
+    AllowedDomains : string list
+    /// Tenants visible to a management company
+    AuthorizedTenants : ApplicationTenantId list
+}
```

Comment:
Interesting. But did you forget to use these types in the test or did you initially intended to use them but then you didn't have to and you forgot them here?

Reply:
You are right! It must be on line 246